website/docs/learning-and-resources/how-to-guides/how-to-use-git-flow-for-different-environments.md

```diff
@@ -0,0 +1,29 @@
+# How to use Git flow for different environments
+
+In this guide, you’ll learn how to create different environments for your App using the Git connection in Appsmith.
+
+## Environments
+If you have an App connected to Git in Appsmith, you can create separate branches to work on it independently. The problem arises when you are working on the same database. Any alteration to the database can cause multiple failures on various branches. So, it's better to have a different environment with a test database for developing an application that won't affect the production database. But, what are Environments?
+Environments are container entities within a space that allows you to create and maintain multiple versions of your App and make changes to them in isolation. A separate environment can be an app with a test database in another workspace or instance.
+
```

Comment:
We do not support GSheet, Twilio, API or any SAAS-type datasource. I guess we should mention this beforehand to reduce the Aforce queries.

Reply:
Added an info tag at the bottom.

Reply:
Let's move the callout right at the top here so users know first hand which datasources it works on